tech/languages/java/java-build-tools-installation.md

```diff
@@ -24,22 +24,46 @@ Last but not least, still quite popular duo [Ant+Ivy](http://ant.apache.org/ivy/
 $ sudo dnf install ant apache-ivy
 ```
 
-## DevAssistant is obsolete, use it for reference only
+## Creating a Maven project
 
-You can get started with Maven by using [DevAssistant](/tools/devassistant/about.html) to create a simple project, simply type (after you have DevAssistant setup and installed):   
+### Using `mvn` command-line tool
+
+Use the command from the [Maven quick start guide](https://maven.apache.org/guides/getting-started/maven-in-five-minutes.html):
```

Comment:
This sounds great! But don't we have the `mvn` packaged in Fedora?

If it's about the guide, please put the reference after the `quick start` "text" instead.

Reply:
Let's address this in standalone PR if it needs to be changed (**is mvn in Fedora?**).